MobileWallet/Screens/AppEntry/Splash/WalletCreationViewController.swift

```diff
@@ -0,0 +1,222 @@
+            self.view.layoutIfNeeded()
+        }) { [weak self] (_) in
+            guard let self = self else { return }
+            self.displaySecondLabelAnimation()
+        }
+    }
+
+    private func displaySecondLabelAnimation() {
+        UIView.animate(withDuration: 1, animations: { [weak self] in
+            guard let self = self else { return }
+            self.secondLabel.alpha = 1.0
+            self.view.layoutIfNeeded()
+        }) { [weak self] (_) in
+            guard let self = self else { return }
+            //self.displaySecondLabelAnimation()
+            DispatchQueue.main.asyncAfter(deadline: .now() + 3) { [weak self] in
+                guard let self = self else { return }
+                self.removeSecondLabelAnimation()
+            }
+        }
+    }
+
+    private func removeSecondLabelAnimation() {
+        UIView.animate(withDuration: 1, animations: { [weak self] in
+            guard let self = self else { return }
+            self.secondLabel.alpha = 0.0
+            self.view.layoutIfNeeded()
+        }) { [weak self] (_) in
+            guard let self = self else { return }
+            let secondLabelString = NSLocalizedString("Awesome!\nNow create your Emoji ID", comment: "Second label on wallet creation")
+            let attributedString = NSMutableAttributedString(string: secondLabelString,
+                                                             attributes: [
+                                                                .font: Theme.shared.fonts.secondLabelSecondText ?? UIFont.systemFont(ofSize: 12),
+                                                                .foregroundColor: Theme.shared.colors.creatingWalletSecondLabel,
+                                                                .kern: -0.33
+            ])
+
+            let splitString = secondLabelString.components(separatedBy: "\n")
+            if splitString.count > 0 {
+                let length = splitString[0].count
+                attributedString.addAttribute(.font,
+                                              value: Theme.shared.fonts.secondLabelFirstText ?? UIFont.systemFont(ofSize: 12),
+                                              range: NSRange(location: 0, length: length))
+
+                self.secondLabel.attributedText = attributedString
+            }
+            DispatchQueue.main.asyncAfter(deadline: .now()) { [weak self] in
+                guard let self = self else { return }
+                self.runCheckMarkAnimation()
+            }
+        }
+    }
+
+    private func runCheckMarkAnimation() {
+        //print("run checkmark animation")
+        loadAnimation()
+        startAnimation()
+    }
+
+    private func loadAnimation() {
+        let animation = Animation.named("CheckMark")
+        animationView.animation = animation
+    }
+
+    private func startAnimation() {
+        #if targetEnvironment(simulator)
+          animationView.animationSpeed = 1
```

Comment:
Can probably remove this. I just added it to the splash screen for existing users to shorten the time it took to test the transaction list